nautobot/docs/development/getting-started.md

```diff
@@ -555,6 +555,15 @@ To enforce best practices around consistent [coding style](style-guide.md), Naut
 | `invoke flake8`         | `flake8`                     |
 | `invoke black`          | `black`                      |
 
+### Handling Migrations
+
+If your branch modifies a Django model (and as a result requires a database schema modification), after running `invoke makemigrations` rename the generated migration files to something more meaningful before pushing. For example `0004_provider_increase_account_length` instead of `0004_auto_20211220_2104`.
```

Comment:
Might be worth mentioning this can be done with the `invoke makemigrations -n/--name` argument? But that is minor.

Reply:
💡 I will document that and get this updated since this is net-new to the docs.

Reply:
@jathanism Updated.